web/assets/javascripts/application.js

```diff
@@ -108,4 +110,5 @@ function replacePage(text) {
   document.querySelector('.status').replaceWith(header_status)
 
   updateFuzzyTimes();
```

Comment:
It looks like `addListensers` already calls `updateFuzzyTimes`, so this is probably now redundant.

Reply:
Fixed. 👍